### ISSUE TITLE
Projector point isolation : Cosine distance change and explained variance decrease

### ISSUE THREAD

**Issue by Sora77** [tensorflow/tensorboard#1210]
Hello,
This issue is a request for an explaination about two points in Tensorboard's projector :
#### 1 - Cosine distance changes when isolating a selection of points
Cosine distance seems to change when zooming on points. The right side column showing the nearest neighours has the mention 'in the original space'. I think I'm not the only one who also interpreted it as 'the cosine distance is also calculated in the original space', so I think it should be mentioned that it is not the case. The cosine distance seems to have been mapped from it's original interval to an interval somewhat close to [0, 2], but I couldn't find the formula for this change anywhere.
To sum it up:
- The change occuring in the formula should be mentioned and documented
- We should have a button to enable/disable this modified cosine distance

#### 2 - PCA displays an explained variance of 100.0%, but it decreases when isolating points
Initial PCA with 100% explained variance shows a superposition of points with a cosine distance value of 0.000. When selecting and isolating these points, the superposition is no more, and the explained variance goes down to 92.6%. The superposed points are 20 in a 450 points representation, and there are many of such superposition cases in the same embedding (hence the fact that 100% variance explained is a bit shady.
[Here](https://drive.google.com/file/d/1zd_251M7BYVftu37jKMnXa3CI9oNAca3/view?usp=sharing) is a link to my Tensorboard files folder.
An example of the case of superposition can be found in the embedding _Embedddings/t_embedding_ if you look for the point labeled _technicien assurance qualité_.

Thank you for your time, and pardon my mistakes if I submitted this first issue in the wrong format.

-------------------------

Tensorboard version : 1.8.0
Tensorflow version: 1.6.0
Python Version : 3.6
OS : Windows 10

**Comment by jart** [comment]
This issue deserves an achievement for most tags. We really appreciate feedback. We're happy to consider feature requests. Support is better directed to [Stack Overflow](http://stackoverflow.com/questions/tagged/tensorboard) since there's a larger community there with better searchability. We'd very much welcome contributions that improve our documentation.

> We should have a button to enable/disable this modified cosine distance

@nsthorat do you have thoughts on this feature request?

**Comment by nfelt** [comment]
I think the question about the meaning of cosine distance here also came up in #2421.

**Comment by Sora77** [comment]
Yep, it answers the first part of the question. Thank your for pointing it out !
